tools/updatePubEdgeDefaults.js

```diff
@@ -0,0 +1,22 @@
+import { Pub } from 'server/models';
```

Comment:
I've been putting data migrations like this in the `migrations` directory. Do you think they should live in their own directory apart from the schema changes instead?

Reply:
I've distinguished the two in the past. Perhaps we have two subdirectories like `migrations/schema` and `migrations/data`.